portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +33,29 @@ function setLogin() {
     }
   });
 }
+
+function createMap() {
+  var cmu = {lat: 40.443378, lng: -79.944404};
+  var map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 18,
+    center: cmu,
+    mapTypeId: 'satellite',
+    tilt: 45
```

Comment:
[style] use trailing commas https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
(and throughout the file)